docs/docs/upgrade-gatsby-and-dependencies.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Upgrade Gatsby for minor or patch releases
+---
+
+## Introduction
+
+To keep up with the latest bug fixes, security patches and minor releases from both Gatsby and its dependencies, you should constantly upgrade to the latest version of each one.
+
+In this guide you will learn how to upgrade Gatsby and its dependencies for minor or patch releases.
+
+## Semantic versioning
+
+As many other pacakages, Gatsby uses [semantic versioning](https://semver.org/) to tag new versions and indicate what kind of changes are introduced in every new release.
+
+This guide is meant to teach you how to upgrade Gatsby for minor or patch releases. For major changes you can refer to our [Release and Migrations](https://www.gatsbyjs.org/docs/releases-and-migration/) reference guide overview for the corresponding guide to upgrade.
```

Comment:
```suggestion
This guide is meant to teach you how to upgrade Gatsby for minor or patch releases. For major changes you can refer to the [Release and Migrations](/docs/releases-and-migration/) reference guide overview for the corresponding guide to upgrade.
```

Updating to align with the [Gatsby Style Guide on using "you" as the pronoun](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun) (I omit pronouns sometimes to make that easier to achieve). We also use relative URLs so they work when building the site locally.